Survey-form HTML code

```diff
@@ -0,0 +1,49 @@
+<h1 id="title">Survey Form</h1>
```

Comment:
make sure this file has a .html extension and include the doctype declaration -> 

```
<!DOCTYPE html>
<html>
<body>
... your html
</body>
</html>
```

Reply:
Ahh ok ok ill do that now